### DISCUSSION TITLE
cache and soft link?

### DISCUSSION THREAD

**Discussion by tbolon** [mastodon/mastodon#20228] [Q&A] [answered]
Is it supported to create a soft link for the /public/system/cache directory?

Or is there any recommended way to quickly move the cache directory to another place on the filesystem? I have an attached disk on another path, and would like to migrate cache data on it.

**Comment by dogrileycom** [comment]
Same question here. I couldn't tell if you can configure it in .env.production with the paperclip settings for local storage. So was thinking of fixing via OS with symlink

**Comment by supernovae** [comment]
Would be nice to be able to define where each volume is and set them up with their own S3/NFS/local mount

**Comment by supernovae** [comment]
I asked a round-about question on this over here: https://github.com/mastodon/mastodon/discussions/20248 👍

**Comment by tbolon** [comment, marked as answer]
I gave up about that, because I was receiving strange `Read-only file system @ dir_s_mkdir`.
But it turns out this solution could work, because this error was related to systemd.

I was able to move the /system directory after some other changes, I will create another topic about that to collect feedback.

**Reply by tbolon** [reply]
I have created a discussion with my solution. #20252
